File: 프로그래머스/lv0/120904. 숫자 찾기/숫자 찾기.java
class Solution {
    public int solution(int num, int k) {
        int answer = 0;
        String [] number = String.valueOf(num).split("");
        String a = String.valueOf(k);
        for (int i = 0; i<number.length; i++) {
            if (number[i].equals(a)) {
                answer = i+1;
                break;
            }
            else {
                answer = -1;
            }
        }
        return answer;
    }
}

File: 프로그래머스/lv0/120904. 숫자 찾기/숫자 찾기.py
def solution(num, k):
    answer = 0
    #문자열로 바꾸면 find()함수로 찾을수 있다  / 같은 숫자가 없다면 자동으로 -1을 반환해준다.
    if str(k) in str(num):
        answer = str(num).find(str(k)) +1
    else:
        answer = str(num).find(str(k))
    return answer